app/ui/src/main/java/de/mm20/launcher2/ui/component/ShapedLauncherIcon.kt

```diff
@@ -250,7 +250,7 @@ fun ShapedLauncherIcon(
                     _badge.progress?.let {
                         val progress by animateFloatAsState(it)
                         CircularProgressIndicator(
-                            modifier = Modifier.fillMaxSize(),
+                            modifier = Modifier.fillMaxSize(0.8f),
```

Comment:
why?

Reply:
Sorry, not intended for this patchset, it was to make clearer a progress circle on notificacion dot separating the progress from the border.